awx_collection/tools/ansible.cfg

```diff
@@ -0,0 +1,2 @@
+[defaults]
+stdout_callback = yaml
```

Comment:
I think maybe you want to deal with this in a separate PR

but it might be good to put this in the `build_ignore` in the galaxy.yml template. I did that with some playbooks for building. We want to ship tests, but my thinking is that anything used for producing the collection shouldn't be shipped as a part of the collection.

Reply:
I agree with this, this tools folder should not exist in the collections distribution.